src/patterns/dosdp-patterns/left_teeth.yaml

```diff
@@ -0,0 +1,91 @@
+pattern_name: right__left_teeth
+pattern_iri: http://purl.obolibrary.org/obo/uberon/patterns/left_teeth.yaml
+
+description: "This pattern defines primary and secondary teeth for the left side of the mouth."
+
+contributors:
+  - https://orcid.org/0000-0001-9625-1899 # Bill Duncan
+
+classes:
+  thing: "owl:Thing"
+  Homo sapiens: "NCBITaxon:9606"
+  calcareous tooth: "UBERON:0001091"
+  jaw region: "UBERON:0011595"
+  maxilla: "UBERON:0002397"
+  mandible: "UBERON:0001684"
+
+relations:
+  in lateral side of: "BSPO:0000126"
+  in right side of: "BSPO:0000121"
+  in left side of: "BSPO:0000120"
+
+annotationProperties:
+  hasExactSynonym: "oboInOwl:hasExactSynonym"
+  present_in_taxon: "RO:0002175"
+  contributor: "http://purl.org/dc/terms/contributor"
+  creation_date: "oboInOwl:creation_date"
+  hasDbXref: "oboInOwl:hasDbXref"
+  hasSynonymType: "oboInOwl:hasSynonymType"
+
+vars:
+  parent_class: "calcareous tooth"       # UBERON:0001091
+  jaw_region: "jaw region"               # UBERON:0011595
+  FMA_label: "xsd:string"                # e.g., right upper third secondary molar tooth
+  human_preferred: "UBERON:core#HUMAN_PREFERRED"
+
+data_vars:
+  laterality: "xsd:string"     # e.g., right, left
+
+data_list_vars:                # separate multiple entries by "|"
+  synonym: "xsd:string"        # e.g., ADA tooth 1, FDI tooth 18
+  creator: "xsd:anyURI"        # e.g., https://orcid.org/0000-0001-9625-1899
+  date_created: "xsd:dateTime" # e.g., 2022-11-13T00:00:00Z; note: this shows up as a string in Protege
+  taxon: "xsd:string"          # e.g., obo:NCBITaxon_9606; note: the full IRI is being used as a value
+  FMA_xref: "xsd:string"       # e.g., FMA:55696
```

Comment:
Move `taxon` to `vars` because it's an OWL class. Change it to `taxon: "Homo sapiens"`, as you defined before. Also move `creator` to `vars` and add value "https://orcid.org/0000-0001-9625-1899" to be an entity in the OWL.
Move `date_created` and `FMA_xref` to `data_vars` as they are a single value.

Reply:
Unfortunately, it's not possible to have other type than `xsd:string`, that's why date_created is generated as string even though you defined as `xsd:dateTime`. I'll create an issue to fix this using SPARQL.

Reply:
done
```
vars:
  parent_class: "calcareous tooth"  # UBERON:0001091
  jaw_region: "jaw region"          # UBERON:0011595
  taxon: "Homo sapiens"             # obo:NCBITaxon_9606
  creator: "xsd:anyURI"             # e.g., https://orcid.org/0000-0001-9625-1899
  human_preferred: "obo:uberon/core#HUMAN_PREFERRED"
```

```
annotationProperty: present_in_taxon
    var: taxon
```
```
annotationProperty: contributor
    var: creator
```

Reply:
Sorry you didn't make the second change I suggested.

> Move date_created and FMA_xref to data_vars as they are a single value.

Reply:
And also, this part

> [move creator to vars] add value "https://orcid.org/0000-0001-9625-1899" to be an entity in the OWL.

Reply:
@anitacaron 
I changed the requested `data_list_vars` to `data_vars`.

> [move creator to vars] add value "https://orcid.org/0000-0001-9625-1899" to be an entity in the OWL.

This has been done. See [here](https://github.com/obophenotype/uberon/blob/15a13a958719695f1b42cf9430bd0efb62bee37a/src/patterns/dosdp-patterns/right_teeth.yaml#L34). 
```
vars:
  parent_class: "calcareous tooth"  # UBERON:0001091
  jaw_region: "jaw region"          # UBERON:0011595
  taxon: "Homo sapiens"             # obo:NCBITaxon_9606
  creator: "xsd:anyURI"             # e.g., https://orcid.org/0000-0001-9625-1899
  human_preferred: "obo:uberon/core#HUMAN_PREFERRED"
```
Since the range is `xsd:anyURI` a URI is output in the `dcterms:contributor` annotation.

Reply:
I was thinking of adding your ORCID as the default value.